internal/boxcli/update.go

```diff
@@ -18,10 +19,10 @@ func updateCmd() *cobra.Command {
 
 	command := &cobra.Command{
 		Use:   "update [pkg]...",
-		Short: "Updates packages in your devbox",
-		Long: "Updates one, many, or all packages in your devbox. " +
+		Short: "Update packages in your devbox",
+		Long: "Update one, many, or all packages in your devbox. " +
 			"If no packages are specified, all packages will be updated. " +
-			"Only updates versioned packages (e.g. `python@3.11`), not packages that are pinned to a nix channel (e.g. `python3`)",
+			"Only update versioned packages (e.g. `python@3.11`), not packages that are pinned to a nix channel (e.g. `python3`)",
```

Comment:
updates is a bit more accurate because it is describing what it does instead of being imperative. (the first 2 uses can be imperative so `update` is ok.)

Either way, this functionality is actually changing in https://github.com/jetpack-io/devbox/pull/1107 so I'll update it there.

Reply:
Then I think it should be `Only versioned packages are updated ...`